oeps/oep-0032-arch-unique-identifier-for-users.rst

```diff
@@ -33,8 +33,12 @@ We will use the *LMS user_id* to uniquely identify a user in the Open edX system
 
 The *LMS user_id* should be used for all internal events. Additionally, the *LMS user_id* can be used for public events, REST APIs, and JavaScript APIs.
 
-.. _Django auth_user: https://docs.djangoproject.com/en/2.0/topics/auth/default/#user-objects
+.. warning::
+
+    A new external user id should still be used with third-parties under certain circumstances. This is especially important when the user id is sent along with other `Personally Identifiable Information (PII)`_. See the `ADR for the edx-platform external_user_ids Django app`_ for more details.
```

Comment:
[clarification] What does "certain circumstances" mean? Are we not recommending the external-user-id for all external uses?

Reply:
I don't feel I have all the answers, or I would help make a more complete update. My intent was to add a temporary improvement to raise awareness that there were additional solutions and situation to consider, which some people were not aware of, because it wasn't listed in the OEP. At a minimum, this might trigger a question and conversation, rather than people presuming incorrectly that they should use the LMS user_id for "certain situations".

I think the LMS user_id is used with Caliper and xAPI. I would consider that external, but I'm not sure what external versus internal means, which is part of what would need to be worked out for a more complete update.

Additionally, I realized that we haven't yet started superseding OEPs. I'm also not clear yet if we think we will be contradicting the original OEP or adding clarity to cases that weren't considered. I know this contradicts what I _thought_ we meant by the OEP.